bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Trajectory/Orbit/Model.cpp

```diff
@@ -41,11 +43,15 @@ inline void                     OpenSpaceToolkitAstrodynamicsPy_Trajectory_Orbit
 
         .def("is_kepler", +[] (const Model& aModel) -> bool { return aModel.is<Kepler>() ; })
         .def("is_sgp4", +[] (const Model& aModel) -> bool { return aModel.is<SGP4>() ; })
+        .def("is_propagated", +[] (const Model& aModel) -> bool { return aModel.is<Propagated>() ; })
 
         // .def("as_kepler", +[] (const Model& aModel) -> const Kepler& { return aModel.as<Kepler>() ; }, return_value_policy<reference_existing_object>())
```

Comment:
same here?

Reply:
@lucas-bremond @Derollez same question, is this something that would be useful to actually try to implement, cause I know the c++ code behind it doesn't exist or is commented out

Reply:
same here